sklearn/metrics/tests/test_classification.py

```diff
@@ -914,6 +914,18 @@ def test_confusion_matrix_multiclass_subset_labels():
                          labels=[extra_label, extra_label + 1])
 
 
+@pytest.mark.parametrize(
+    'labels', (None, [0, 1], [0, 1, 2]),
```

Comment:
```suggestion
    'labels', ([], [0, 1], [0, 1, 2]),
```

Reply:
I don't really think labels=[] makes sense or should be permitted were we strict about it

Reply:
Yes, we were and I think we should keep this behaviour.

```python
In [2]: confusion_matrix([], [], labels=[])                                                                                                                                 
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-fb871fbb6d6b> in <module>
----> 1 confusion_matrix([], [], labels=[])

~/Documents/code/toolbox/scikit-learn/sklearn/metrics/_classification.py in confusion_matrix(y_true, y_pred, labels, sample_weight, normalize)
    278         labels = np.asarray(labels)
    279         if np.all([l not in y_true for l in labels]):
--> 280             raise ValueError("At least one label specified must be in y_true")
    281 
    282     if sample_weight is None:

ValueError: At least one label specified must be in y_true
```